packages/ember-htmlbars/tests/helpers/closure_component_test.js

```diff
@@ -1,776 +0,0 @@
-  }).create();
-
-  runAppend(component);
-  equal(component.$().text(), 'Hodi Hodari', '-looked-up component rendered');
-});
-
-QUnit.test('renders with component helper with curried params, hash', function() {
-  let LookedUp = Component.extend();
-  LookedUp.reopenClass({
-    positionalParams: ['name']
-  });
-  owner.register(
-    'component:-looked-up',
-    LookedUp
-  );
-  owner.register(
-    'template:components/-looked-up',
-    compile(`{{greeting}} {{name}}`)
-  );
-
-  let template = compile(
-    `{{component (component "-looked-up" "Hodari" greeting="Hodi") greeting="Hola"}}`
-  );
-  component = Component.extend({
-    [OWNER]: owner,
-    template
-  }).create();
-
-  runAppend(component);
-  equal(component.$().text(), 'Hola Hodari', '-looked-up component rendered');
-});
-
-QUnit.test('updates when component path is bound', function() {
-  let Mandarin = Component.extend();
-  owner.register(
-    'component:-mandarin',
-    Mandarin
-  );
-  owner.register(
-    'template:components/-mandarin',
-    compile(`ni hao`)
-  );
-  owner.register(
-    'template:components/-hindi',
-    compile(`Namaste`)
-  );
-
-  let template = compile('{{component (component lookupComponent)}}');
-  component = Component.extend({
-    [OWNER]: owner,
-    template,
-    lookupComponent: '-mandarin'
-  }).create();
-
-  runAppend(component);
-
-  equal(component.$().text(), `ni hao`,
-        'mandarin lookupComponent renders greeting');
-  run(() => {
-    component.set('lookupComponent', '-hindi');
-  });
-  equal(component.$().text(), `Namaste`,
-        'hindi lookupComponent renders greeting');
-});
-
-QUnit.test('updates when curried hash argument is bound', function() {
-  owner.register(
-    'template:components/-looked-up',
-    compile(`{{greeting}}`)
-  );
-
-  let template = compile(
-    `{{component (component "-looked-up" greeting=greeting)}}`
-  );
-
-  component = Component.extend({
-    [OWNER]: owner,
-    template
-  }).create();
-
-  runAppend(component);
-  equal(component.$().text(), '', '-looked-up component rendered');
-  run(() => {
-    component.set('greeting', 'Hodi');
-  });
-  equal(component.$().text(), `Hodi`,
-        'greeting is bound');
-});
-
-QUnit.test('updates when curried hash arguments is bound in block form', function() {
-  owner.register(
-    'template:components/-looked-up',
-    compile(`{{greeting}}`)
-  );
-
-  let template = compile(
-    `{{#with (hash comp=(component "-looked-up" greeting=greeting)) as |my|}}
-      {{#my.comp}}{{/my.comp}}
-    {{/with}}`
-  );
-
-  component = Component.extend({
-    [OWNER]: owner,
-    template
-  }).create();
-
-  runAppend(component);
-  equal(component.$().text().trim(), '', '-looked-up component rendered');
-  run(() => {
-    component.set('greeting', 'Hodi');
-  });
-  equal(component.$().text().trim(), `Hodi`,
-        'greeting is bound');
-});
-
-QUnit.test('nested components overwrites named positional parameters', function() {
```

Comment:
This [one](https://github.com/emberjs/ember.js/pull/13214/files#diff-8e2359cbe9c551044787f512fe265932R118)